ui/components/app-wrapper.js

```diff
@@ -19,13 +19,20 @@ Router.onRouteChangeError = NProgress.done;
 Router.onRouteChangeStart = NProgress.start;
 
 
-export default function wrapPage(Page, dataFn) {
+export default function wrapPage(Page, dataFn, headerFooterParams = { showSearch: true }) {
```

Comment:
Unfortunately, default values are shared across all calls to `wrapPage`. Combine this with a mutable value and we'll get very unexpected results.

We can get around that with something like:
```js
export default function wrapFunction(Page, dataFn, headerFooterParams) {
  const hfParams = headerFooterParams || { showSearch: true };
```